unittests/test_lib_ogl.py

```diff
@@ -0,0 +1,83 @@
+import imp_unittest, unittest
+import wtc
+import wx
+
+import wx.lib.ogl as ogl
+
+
+
+class lib_ogl_Tests(wtc.WidgetTestCase):
+
+    def test_lib_oglCtor(self):
+        ogl.OGLInitialize()
+        osc = ogl.ShapeCanvas(self.frame)
+        self.diagram = ogl.Diagram()
+        osc.SetDiagram(self.diagram)
+        self.diagram.SetCanvas(osc)
+        
+        aShape = ogl.RectangleShape(w=50, h=50)
+        aShape.SetCanvas(osc)
+        self.diagram.AddShape(aShape)
+
+        region1 = ogl.ShapeRegion()
+        region1.SetText('DividedShape')
+        region1.SetProportions(0.0, 0.2)
+        region1.SetFormatMode(ogl.FORMAT_CENTRE_HORIZ)
+        aShape.AddRegion(region1)
+        
```

Comment:
It would be nice to create one of every type of Shape for this test so we can at least be sure that they are all constructable.